ios-target-app/Common/Models/Avatar.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Avatar.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 1/11/22.
+//
+
+import Foundation
+
+struct Avatar: Codable {
+  let originalUrl, normalUrl, smallThumbUrl: String?
```

Comment:
A couple of things here:
1. Please use the convention from the [style guide](https://rootstrap.github.io/swift/#type-variable-and-function-declarations).
2. For acronyms use uppercase letters, i.e #6 
3. URL is a Decodable type so you could use that type directly

```suggestion
  let originalUrl: URL?
  let normalURL: URL?
  let smallThumbURL: URL?
```

Reply:
don't forget to update the coding keys below as well.